R/run.R

```diff
@@ -679,16 +679,19 @@ view_run_details <- function(run, auto_refresh = TRUE) {
 
     # stop and remove any existing widget job before submitting script
     # nolint start
```

Comment:
With this change, I assume we can remove $nolint directives.

Reply:
Actually, no. lintr also produces errors because the rstudioapi function is camel case (jobRunScript) while the rest of our variables are snake case.

Reply:
Gotcha. Thanks!